jobs/forms.py

```diff
@@ -19,3 +19,18 @@ class Meta:
             'title': 'Job title'
         }
 
+
+class MeetupForm(forms.ModelForm):
+
+    class Meta:
+        model = Meetup
+        exclude = ['reviewer', 'review_status', 'reviewers_comment',
+            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        ]
+
```

Comment:
@MarysiaLowas You approached the company in the jobs form a bit differently - was there a reason we didn't use a drop down for the company there? Could companies possibly not want themselves listed in a drop down list of all companies that have jobs available (since they'd all be visible on the site, I don't suppose it should be a problem).  Either way, I guess the two of you should take a look at each other's solutions and let's make them consistent across jobs and meetups.